BCStudentSoftwareDevTeam/celts
# Table for issue 78

This PR is for issue #78 
Issue was to create a page where instructors can see and modify their courses, a.k.a. the following wireframe: https://balsamiq-wireframes.appspot.com/?state={%22action%22:%22open%22,%22ids%22:[%221ziGyTzz6J7iHjyzmsUzj0yzvuFdITg3n%22],%22userId%22:%22114880022803373311847%22} 

In addition to the table, we also implemented a working withdraw function. However, "renew," "edit," and "view" are dependent on pages not yet created. So they are blank. 

Url:
/courseProposals 

To test: 
Go to the url and select "Withdraw" from the select box. The course should be removed. 
Also, go to the file test_proposals.py and have at it.
